x/marker/types/codec.go

```diff
@@ -12,21 +14,11 @@ import (
 
 // RegisterInterfaces registers implementations for the tx messages
 func RegisterInterfaces(registry types.InterfaceRegistry) {
-	registry.RegisterImplementations((*sdk.Msg)(nil),
-		&MsgAddMarkerRequest{},
-		&MsgAddAccessRequest{},
-		&MsgDeleteAccessRequest{},
-		&MsgFinalizeRequest{},
-		&MsgActivateRequest{},
-		&MsgCancelRequest{},
-		&MsgDeleteRequest{},
-		&MsgMintRequest{},
-		&MsgBurnRequest{},
-		&MsgWithdrawRequest{},
-		&MsgTransferRequest{},
-		&MsgIbcTransferRequest{},
-		&MsgSetDenomMetadataRequest{},
-	)
+	messages := make([]proto.Message, len(allRequestMsgs))
+	for i, msg := range allRequestMsgs {
+		messages[i] = msg
+	}
+	registry.RegisterImplementations((*sdk.Msg)(nil), messages...)
```

Comment:
I like this a lot.  Since, adding a change to both places sometimes gets missed.